mobile-app/app/screens/AppNavigator/screens/Auctions/screens/ConfirmPlaceBidScreen.tsx

```diff
@@ -93,110 +96,158 @@ export function ConfirmPlaceBidScreen(props: Props): JSX.Element {
 
-      <SubmitButtonGroup
-        label={translate("screens/ConfirmPlaceBidScreen", "CONFIRM PLACE BID")}
-        isDisabled={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
-        isProcessing={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
-        processingLabel={translate(
-          "screens/ConfirmPlaceBidScreen",
-          getSubmitLabel()
-        )}
-        onCancel={onCancel}
-        displayCancelBtn
-        onSubmit={onSubmit}
-        title="bid"
-      />
-    </ThemedScrollView>
+        <TextRowV2
+          containerStyle={{
+            style: tailwind("flex-row items-start w-full bg-transparent mt-6"),
+            ...containerThemeOptions,
+          }}
+          lhs={{
+            value: translate("screens/ConfirmPlaceBidScreen", "Vault ID"),
+            testID: "text_vault_id_label",
+            themedProps: lhsThemedProps,
+          }}
+          rhs={{
+            value: vault.vaultId,
+            testID: "text_vault_id",
+            numberOfLines: 1,
+            ellipsizeMode: "middle",
+            themedProps: rhsThemedProps,
+          }}
+        />
+        <TextRowV2
+          containerStyle={{
+            style: tailwind("flex-row items-start w-full bg-transparent mt-6"),
+            ...containerThemeOptions,
+          }}
+          lhs={{
+            value: translate("screens/ConfirmPlaceBidScreen", "Vault owner ID"),
+            testID: "text_vault_owner_label",
+            themedProps: lhsThemedProps,
+          }}
+          rhs={{
+            value: vault.ownerAddress,
+            testID: "text_vault_owner_id",
+            numberOfLines: 1,
+            ellipsizeMode: "middle",
+            themedProps: rhsThemedProps,
+          }}
+        />
+        <NumberRowV2
+          containerStyle={{
+            style: tailwind(
+              "flex-row items-start w-full bg-transparent border-b-0.5 pb-5 mt-6"
+            ),
+            ...containerThemeOptions,
+          }}
+          lhs={{
+            value: translate(
+              "screens/ConfirmPlaceBidScreen",
+              "Liquidation height"
+            ),
+            testID: "text_liquidation_height_label",
+            themedProps: lhsThemedProps,
+          }}
+          rhs={{
+            value: vault.liquidationHeight,
+            testID: "text_liquidation_height",
+            themedProps: rhsThemedProps,
+          }}
+        />
+
+        <View style={tailwind("pt-14 px-7")}>
+          <ThemedTextV2
+            light={tailwind("text-mono-light-v2-500")}
+            dark={tailwind("text-mono-dark-v2-500")}
+            style={tailwind("text-center text-xs font-normal-v2")}
+          >
+            {translate(
+              "screens/ConfirmPlaceBidScreen",
+              "Amount will be deducted from your current wallet"
+            )}
+          </ThemedTextV2>
+          <SubmitButtonGroupV2
+            isDisabled={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
```

Comment:
can we add condition to disable the button when auction time ended too?

Reply:
@kyleleow should we display "Auction timeout" error on confirm screen as well? Or just disable the button

Reply:
yea let's use the same error msg as the screen before